.github/workflows/swift.yml

```
@@ -0,0 +1,15 @@
+name: Swift
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: swift build -v
```

Optional, although it might be worth considering an explicit timeout on the build steps.  If we get a bad runner or GitHub Actions is having an incident, sometimes builds can take forever and waste CI minutes.  Setting an explicit timeout creates some guardrails to keep the steps from running for too long.

The downside is that over time, we occasionally might have to increase the timeout as our codebase grows and builds take longer.

I chose the current suggested timeouts based on the run time of the action, plus some buffer.

```suggestion
    - name: Build
      timeout-minutes: 10
      run: swift build -v
```